tests/ui_tests/app/page/job_list/page.py

```diff
@@ -9,16 +9,126 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from dataclasses import dataclass
+from typing import TypeVar, Union
 
+import allure
+
+from enum import Enum
+from selenium.webdriver.remote.webelement import WebElement
+
+from tests.ui_tests.app.helpers.locator import Locator
 from tests.ui_tests.app.page.common.base_page import (
     BasePageObject,
     PageHeader,
     PageFooter,
 )
+from tests.ui_tests.app.page.common.header import AuthorizedHeaderLocators
+from tests.ui_tests.app.page.common.table.page import CommonTableObj
+from tests.ui_tests.app.page.common.tooltip_links.locator import CommonToolbarLocators
+from tests.ui_tests.app.page.job_list.locators import TaskListLocators
+
+
+class JobStatus(Enum):
+    RUNNING = 'running'
+    SUCCESS = 'success'
+    FAILED = 'failed'
+
+
+@dataclass
+class PopupTaskInfo:
+    """Info about the job from popup"""
+
+    action_name: str
+    status: JobStatus
+
+
+@dataclass
+class TableTaskInfo(PopupTaskInfo):
+    """Info about the job from table row"""
+
+    invoker_objects: str
+    start_date: str
+    finish_date: str
+
+
+TaskInfo = TypeVar('TaskInfo', bound=Union[PopupTaskInfo, TableTaskInfo])
 
 
 class JobListPage(BasePageObject):
     def __init__(self, driver, base_url):
         super().__init__(driver, base_url, "/task")
         self.header = PageHeader(self.driver, self.base_url)
         self.footer = PageFooter(self.driver, self.base_url)
+        self.table = CommonTableObj(self.driver, self.base_url, TaskListLocators.Table)
+
+    def get_task_info_from_table(
+        self, row_num: int = 0, *, full_invoker_objects_link: bool = False
```

Comment:
what is * ?

Reply:
To make arguments after this mark keyword only. Because if something like `get_task_info_from_table(0, True)` will look very strange and not obvious.